data/countries/de/ni/aurich-norden/data.yaml

```
@@ -16,17 +18,21 @@ district: Aurich-Norden
 city: Aurich
 urls:
   - type: WEBSITE
-    url: http://www.gruene-ov-aurich.de/
+    url: https://www.gruene-aurich.org/
+  - type: FACEBOOK
+    url: https://www.facebook.com/GrueneAurich
+emails:
+  - address: info@gruene-aurich.de
 ---
 type: REGIONAL_CHAPTER
 level: DE:ORTSVERBAND
 country: DE
 state: Niedersachsen
 district: Aurich-Norden
-city: Brookmerland/Hinte/ Südbrookmerland
+city: Brookmerland, Hinte, Südbrookmerland
 urls:
   - type: WEBSITE
-    url: http://www.gruene-brookmerland-hinte-suedbrookmerland.de/
+    url: https://www.gruene-bhs.de/
```

```suggestion
    url: http://www.gruene-bhs.de/
```

The HTTPS site delivers a bad certificate, so I assume they want `http://gruene-bhs.de/` to be used.